src/components/tree/phyloTree/helpers.js

```diff
@@ -99,6 +103,51 @@ export const setDisplayOrder = (nodes) => {
   nodes[0].displayOrderRange = [undefined, undefined];
 };
 
+/**
+ * @param {Array<PhyloNode>} nodes
+ * @param {boolean} focus
+ * @returns fn to return a display order (y position) for a node
+ */
+function getDisplayOrderCallback(nodes, focus) {
+  /**
+   * Start at 0 and increase with each node.
+   * Note that this value is shared across invocations of the callback.
+   */
+  let displayOrder = 0;
+
+  /**
+   * Keep track of whether the previous node was selected
+   */
+  let previousWasVisible;
+
+  if (focus) {
+    const numVisible = nodes.filter((d) => !d.hasChildren && d.visibility === NODE_VISIBLE).length;
+    const yProportionFocused = Math.max(0.8, numVisible / nodes.length);
```

Comment:
[minor - not blocking] When we only have one (or a few) tips selected, 80% is a bit much. I played with
```js
    const focusMaxProportion = Math.min(0.8, 0.2 * numVisible)
    const yProportionFocused = Math.max(focusMaxProportion, numVisible / nodes.length);
```
which felt better, but there's probably no perfect solution here.

Reply:
Nice one! Resolving this, but note [the above requested change to this code](https://github.com/nextstrain/auspice/pull/1373#discussion_r1807940275)